src/js/components/detachedDashboard/DetachedDashboardContent.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+  * DetachedDashboardContent.jsx
+  * Created by Daniel Boos 6/29/17
+  **/
+
+import React from 'react';
+import DetachedDashboardTable from './DetachedDashboardTable.jsx';
+
+export default class DetachedDashboardContent extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
Does this need to hold the page and certified page? all the page logic is handled within table so this seems redundant